client/src/components/PropertiesList.tsx

```diff
@@ -61,6 +62,16 @@ const FilterContextProvider: React.FC<{}> = (props) => {
   return <FilterContext.Provider value={useValue()}>{props.children}</FilterContext.Provider>;
 };
 
+export type LogPortfolioAnalytics = (
```

Comment:
thoughts on renaming the type to `PortfolioAnalyticsEvent`? since the `logPortfolioAnalytics` is also the function that actually logs the event

Reply:
oh yeah, good point - much more clear